server/src/main/java/org/apache/uniffle/server/ShuffleServerGrpcService.java

```diff
@@ -186,6 +186,16 @@ public void sendShuffleData(SendShuffleDataRequest req,
     String appId = req.getAppId();
     int shuffleId = req.getShuffleId();
     long requireBufferId = req.getRequireBufferId();
+    long sendTime = req.getTimestamp();
+    if (sendTime > 0) {
+      /*
+      * Here we record the transport time, but we don't consider the impact of data size on transport time.
+      * The amount of data will not cause great fluctuations in latency. For example, 100K costs 1ms,
+      * and 1M costs 10ms. This seems like a normal fluctuation, but it may rise to 10s when the server load is high.
+      * */
+      shuffleServer.getGrpcMetrics().recordTransportTime(ShuffleServerGrpcMetrics.SEND_SHUFFLE_DATA_METHOD,
```

Comment:
`System.currentTimeMills() - sendTime` may be less than 0, because they are generated from different machines.

Reply:
Whether does the negative number influence our metrics?

Reply:
Perhaps we can add a comment stating that the time of the client machine and the server machine should be in sync. For the case of less than 0, we do a judgment filter, but time out of sync will also affect metrics

Reply:
> Perhaps we can add a comment stating that the time of the client machine and the server machine should be in sync. For the case of less than 0, we do a judgment filter, but time out of sync will also affect metrics

OK. actually we should have a document to tell users that how to use the metrics and what to notice. But we don't have such documents, so we can only add some comments.